### ISSUE TITLE
Txn version not supported

### ISSUE THREAD

**Issue by James-Sangalli** [Zondax/ledger-kusama#104]
Kusama app v5.30.1
polkadot{.js} extension 0.40.1

I was trying to change the validators I am using for staking but was hit with the message `Txn version not supported`

**Comment by tsiitd** [comment]
the ledger-polkadot issue is resolved after the new ledger polkadot app was released. There is no Kusama updated app on ledger so the Kusama issue still persists

**Comment by jleni** [comment]
yes, Kusama had another additional release and we are working with Ledger on that at the moment.

**Comment by KSM-stuck** [comment]
New Ledger Live update... might help but I doubt it.

**Comment by loupiote** [comment]
@jleni 

for cases like this one i.e. people locked out of their funds if they use the ledger, PLEASE add support for the BIP39 passphrase in https://polkadot.js.org/ , so that at least we do have an emergency recovery option to access our funds. Currently people who use the BIP39 passphrase have no recovery option at all :( :( :(

**Comment by stevetblside** [comment]
Ok @jleni Im having the same issue. So to clarify, you guys are working on the fix for the "txn version not supported" for KSM via ledger?

Feeling uneasy about being stuck

Thank you


![IMG_4407](https://user-images.githubusercontent.com/76492869/138351491-407482ee-dce6-47d1-873b-b587c994c473.jpg)

**Comment by jleni** [comment]
automatically closed
Reopening until Ledger publishes the update

**Comment by loupiote** [comment]
@jleni

> however, we have not been asked to build a user interface for this because of the inherent risk of leaking the mnemonic that is connected to other coins.

Yet you support entering the bip39 24 word menemonic.  Why not implement bip39 completely? There are software wallets that support bip39 mnemonic and passphrase, e.g coinomi. The issue is not about risking to expose a hardware wallet seed. The issue is being able to use a seed thar uses a bip39 passphrase, whether that seed was used in a hardware or software wallet is irrelevant.

**Comment by streetindex** [comment]
Hello,  I am trying to vote on a proposal.

LedgerLive v2.34.3
Ledger Firmware v2.0.0
Kusama App v7.9111.0

Is it an issue with batch calls?  On polkadot.js I've tried connecting ledger via WebUSB as well as WebHID.  I've also tried using Ledger Live's Kusama app as well as their KusamaXL app.  After a couple days now trying here are the errors I've received:

Unable to claim interface
Not supported
Democracy.vote not supported
